wireless/bluetooth/bt_hcicore.c

```diff
@@ -1861,7 +1848,6 @@ int bt_hci_cmd_send_sync(uint16_t opcode, FAR struct bt_buf_s *buf,
       bt_buf_release(buf->u.hci.sync);
     }
 
-  bt_buf_release(buf);
```

Comment:
This `release` was in correct place for the 'taking ownership' function.  But there is also a missing `addref` when we make a strong ref pointer when passing the buffer to `g_btdev.tx_queue`.